vision-quickstart/app/src/main/java/com/google/mlkit/vision/demo/VisionProcessorBase.java

```diff
@@ -88,8 +89,19 @@
     private ByteBuffer processingImage;
     @GuardedBy("this")
     private FrameMetadata processingMetaData;
-    private ArrayList<Pair<Bitmap, Long>> bufferImageList = new ArrayList<>();
-    private Integer depth = 30;
+
+    class bufferImage
+    {
+        Bitmap image;
+        ImageInfo info;
```

Comment:
Any reason you are saving the info as well as the bitmap?  You don't seem to be using it in the code.

Reply:
Yes, it will be used to get rotation degree of image viz. used in text recoginition.
Working locally on it, that's why it's not used here.